package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "status-vac",
+  "version": "1.0.0",
+  "main": "gulpfile.babel.js",
+  "devDependencies": {
+    "autoprefixer": "10.3.1",
+    "babel-preset-env": "1.7.0",
+    "babel-register": "6.26.0",
+    "browser-sync": "2.27.5",
+    "cross-env": "7.0.3",
+    "cross-spawn": "7.0.3",
+    "cssnano": "5.0.7",
+    "gulp": "4.0.2",
+    "gulp-imagemin": "^7.1.0",
+    "gulp-postcss": "9.0.0",
+    "postcss": "8.3.6",
+    "postcss-import": "14.0.2",
+    "tailwindcss": "^2.2.7"
+  },
+  "scripts": {
+    "build:production": "cross-env NODE_ENV=production gulp",
+    "build:dev": "cross-env NODE_ENV=development gulp",
+    "build": "npm run build:production",
+    "dev": "cross-env NODE_ENV=development gulp serve",
+    "start": "npm run dev",
+    "clean": "git clean -fdx",
+    "deploy": "node scripts/deploy.js"
+  },
+  "author": "Ethworks",
```

Comment:
Does this mean something specific here? Ideally we keep branding Vac, and then we can mention in README that site was designed by Ethworks

Also missing LICENSE field

Reply:
sure, thanks